cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -15,6 +16,11 @@
 {% block content_main %}
     <div class="block
                 block__flush-top">
+
+        <div class="block">
```

Comment:
I don't see any issues across responsive sizes, but it would be clearer not nested. Removed in https://github.com/cfpb/cfgov-refresh/commit/92549aa34a68f4fe4c5ed0b1cb6f05805948f98e